docs/reference/stochastic-fictitious-play.rst

```diff
@@ -0,0 +1,99 @@
+.. _stochastic-fictitious-play:
+
+Stochastic fictitious play
+==========================
+
+The stochastic fictitious play algorithm implemented in :code:`Nashpy` is based on the
+one described in [Hofbauer2002]_.
+
+For a game :math:`(A, B)\in\mathbb{R}^{m\times n}` define
+:math:`\kappa_t^{i}:S^{-1}\to\mathbb{N}` to be a function that in a given time
+period :math:`t` for a player :math:`i` maps a strategy :math:`s` from the
+opponent's strategy space :math:`S^{-1}` to a number of total times the opponent
+has played :math:`s`.
+
+As per standard fictitious play, each player assumes their opponent is playing a mixed strategy 
```

Comment:
Can we add a link to the reference docs for standard fictitious play.

Reply:
Nice suggestion. Have added.